scripts/01-prep-aws-objects.sh

```diff
@@ -1,12 +1,18 @@
 #!/bin/bash -e
 
-clusterawsadm alpha bootstrap create-stack
+AWS_ACCOUNT_ID=$(yq r params.yaml aws.account-id)
+./externalDNS/aws/create-externaldns-policy.sh
 
 export AWS_REGION=$(yq r params.yaml aws.region)
 export AWS_ACCESS_KEY_ID=$(yq r params.yaml aws.access-key-id)
 export AWS_SECRET_ACCESS_KEY=$(yq r params.yaml aws.secret-access-key)
-TKG_ENVIRONMENT_NAME=$(yq r params.yaml environment-name)
-SSH_KEY_FILE_NAME=$TKG_ENVIRONMENT_NAME-ssh.pem
+
+clusterawsadm alpha bootstrap create-stack --extra-controlplane-policies arn:aws:iam::$AWS_ACCOUNT_ID:policy/AllowExternalDNSUpdates \
+--extra-node-policies arn:aws:iam::$AWS_ACCOUNT_ID:policy/AllowExternalDNSUpdates
+
+TKG_ENVIRONMENT_NAME=$(yq r params.yaml management-cluster.environment-name)
```

Comment:
Watch out here, you are undoing fixes we just made. I think you need to pull from master and merge into your branch to fix these and maybe others